lib/types.ts

```diff
@@ -10,26 +10,26 @@ export interface SbSvelteSDKOptions extends SbSDKOptions {
 }
 
 export type {
-  AlternateObject,
-  Richtext,
-  RichtextInstance,
+  ISbAlternateObject,
+  ISbRichtext,
+  RichTextResolver,
   SbBlokData,
   SbBlokKeyDataTypes,
   SbSDKOptions,
-  Stories,
-  StoriesParams,
-  Story,
-  StoryData,
-  StoryParams,
+  ISbStories,
+  ISbStoriesParams,
+  ISbStory,
+  ISbStoryData,
+  ISbStoryParams,
   StoryblokBridgeConfigV2,
   StoryblokBridgeV2,
-  StoryblokCache,
-  StoryblokCacheProvider,
+  ISbCache,
+  // StoryblokCacheProvider,
   StoryblokClient,
   StoryblokComponentType,
-  StoryblokConfig,
-  StoryblokManagmentApiResult,
-  StoryblokResult,
+  ISbConfig,
+  ISbManagmentApiResult,
+  ISbResult,
   apiPlugin,
```

Comment:
I think `apiPlugin` should be a normal export

Reply:
good point, thank you